.github/workflows/k8s-tests.yml

```diff
@@ -85,6 +85,7 @@ jobs:
                ./helm/defectdojo \
                --set django.ingress.enabled=true \
                --set imagePullPolicy=Never \
+               --set initializer.keepSeconds="x" \
```

Comment:
Is this value intentional? Docs suggest this `ttlSecondsAfterFinished` should be an integer - I'm not sure what it would do with a value of "x"?

https://github.com/DefectDojo/django-DefectDojo/blob/3f21fb35724788cb87cb3169b781c84e051b6d34/helm/defectdojo/templates/initializer-job.yaml#L22

![Screenshot 2025-02-03 at 17 50 12](https://github.com/user-attachments/assets/7cda1301-9a28-41a3-933f-492c1bbdf856)

Reply:
You would be right if the value were passed to `ttlSecondsAfterFinished`. However thanks to the previous line whole `ttlSecondsAfterFinished` is dropped if you put their value other than a positive integer. This functionality was added in #11257
https://github.com/DefectDojo/django-DefectDojo/blob/3f21fb35724788cb87cb3169b781c84e051b6d34/helm/defectdojo/templates/initializer-job.yaml#L21

I might change it to `keep me`, `0`, `-1`, ... or anything more descriptive here. I'm open to any recommendation.

Reply:
Ah, I should have read up one line! :joy: I think -1 is a pretty common pattern for "ignore this".